actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -9,10 +9,10 @@ class InstallGenerator < ::Rails::Generators::Base
       source_root File.expand_path("templates", __dir__)
 
       def install_javascript_dependencies
-        rails_command "app:update:bin", inline: true
+        rails_command "app:update:bin:yarn", inline: true
```

Comment:
The task namespace here doesn't seem quite right. Especially since we won't update `bin/yarn` if it's already there 😄

How about `app:binstub:yarn`?

Reply:
Yeah, I agree that it may be confusing. I was basing it on the current task, which is `app:update:bin` (but in this case bin refers to the folder). Let's move it a `binstub` namespace 👍.